app/assets/javascripts/sessions.coffee

```diff
@@ -1,3 +1,10 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+ $ ->
```

Comment:
No need to create this method in each coffeescript file.  I'd say keep one copy here in sessions.coffee since sessions are used everywhere.  If you check the source, You'll see that there are now 7 of these functions in the code base.